packages/test/test-utils/tsconfig.json

```diff
@@ -5,7 +5,7 @@
 		"rootDir": "./src",
 		"outDir": "./dist",
 		"composite": true,
-		"types": ["mocha", "node"],
+		"types": ["node", "mocha"],
```

Comment:
Any reason for this vs. alphabetical?

Reply:
victim of repeated refactoring... will revert.